pom.xml

```diff
@@ -67,14 +67,14 @@
   <repositories>
     <repository>
       <id>repo.jenkins-ci.org</id>
-      <url>http://repo.jenkins-ci.org/public/</url>
+      <url>https://repo.jenkins-ci.org/public/</url>
```

Comment:
These don't work at either http or https but it's better convert them to https anyway.

Reply:
WDYM? they are used by the publisher plugin, not meant to be accessible directly in the browser.

Reply:
Browsing at the virtual repo base URL is prohibited for performance reasons, e.g. https://repo.jenkins-ci.org/public/org/ works.

Reply:
Makes sense. Thanks.